src/components/Footer/Footer.tsx

```diff
@@ -1,5 +1,29 @@
 import './Footer.scss';
 
 export const Footer = () => {
-  return <footer>FOOTER COMPONENT</footer>;
+  return (
+    <footer className="footer">
+      <div className="footer__content">
+        <a href="#" className="footer__logo link">
+        </a>
+
+        <nav className="footer__nav nav">
+          <a href="#" className="footer__nav-link">
```

Comment:
add this address to href attribute - https://github.com/fs-mar24-codemafia/react_phone-store 
and target=”_blank” attr to open in new tab